### PR TITLE
Added support for utf8, Spark columns reordering

### PR DESCRIPTION
**Pull request by TanYongSql**

### CODE REVIEW

File: etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/RestppResultSet.java [hunk 1]
```diff
@@ -139,10 +147,13 @@ private void parseKeyValue(ArrayList<Attribute> attribute_list,
...
-      attribute_list.add(new Attribute(key, "INT", Boolean.FALSE));
-      scale_list.add(0);
-    } else if (value instanceof Double) {
+    /**
+     * An attribute's values could be int for some vertices,
+     * and float for others (e.g., pagerank score).
+     * As a workaround, all integers are treated as float.
+     * Will change it back after supporting retrieving query schema.
+     */
+    if ((value instanceof Double) || (value instanceof Integer)) {
```
Comment (gaolk) [inline etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/RestppResultSet.java]: At line 161, do we need to consider negative double value?

File: etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/driver/QueryParser.java [hunk 2]
```diff
@@ -762,14 +772,14 @@ public HttpRequestBase buildQuery(String host, Integer port, Boolean secure,
...
-            StringEntity payload = new StringEntity(json, ContentType.TEXT_PLAIN);
-            post.setEntity(payload);
-          } else {
-            StringEntity payload = new StringEntity(json, ContentType.APPLICATION_JSON);
-            post.setEntity(payload);
+          StringEntity payload = new StringEntity(json, "UTF-8");
+          if ("".equals(this.job)) {
+            payload.setContentType("application/json");
           }
+          payload.setContentEncoding("UTF-8");
```
Comment (gaolk) [inline etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/driver/QueryParser.java]: L779(780) and L782/3: I saw the first is payload and 2nd is "this.payload". Are they the same?

File: etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/RestppResultSet.java (hunk 1 above)
Comment (TanYongSql) [reply etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/RestppResultSet.java]: there's no difference for negative double values, it's used to record how many digits after "."

File: etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/driver/QueryParser.java (hunk 2 above)
Comment (TanYongSql) [reply etl/tg-jdbc-driver/tg-jdbc-restpp/src/main/java/com/tigergraph/jdbc/driver/QueryParser.java]: They are not the same. "this.payload" is a class member variable, "payload" is a local variable. Will rename the former one in next release.

**Pull request merged**